docs/tutorials/add-oauth2-provider.md

```diff
@@ -185,42 +206,18 @@ You can test your OAuth2 provider by trying to login using the [OAuth2 method](h
 
-## 4. Raise a pull request
+If everything goes well, raise both the pull requests and be ready to respond to any feedback which can arise during our code review.
 
-First of all, commit the changes with the message `Added XXX OAuth2 Provider` and push it. This will publish a new branch to your forked version of Appwrite. If you visit it at `github.com/YOUR_USERNAME/appwrite`, you will see a new alert saying you are ready to submit a pull request. Follow the steps GitHub provides, and at the end, you will have your pull request submitted.
+## 5. Raise a pull request
+
+First of all, commit the changes with the message `Added XXX OAuth2 Provider` and push it. This will publish a new branch to your forked version of Appwrite and the Console. If you visit it at `github.com/YOUR_USERNAME/appwrite`, `github.com/YOUR_USERNAME/appwrite/console` you will see a new alert saying you are ready to submit a pull request. Follow the steps GitHub provides, and at the end, you will have your pull request submitted.
 
 ## 🤕 Stuck ?
 
 If you need any help with the contribution, feel free to head over to [our Discord channel](https://appwrite.io/discord) and we'll be happy to help you out.
 
 ## 😉 Need more freedom
 
-If your OAuth provider requires special configuration apart from `clientId` and `clientSecret` you can create a custom form. Currently this is being realized through putting all custom fields as JSON into the `clientSecret` field to keep the project API stable. You can implement your custom form following these steps:
-
-1. Add your custom form in `app/views/console/users/oauth/[PROVIDER].phtml`. Below is a template you can use. Add the filename to `app/config/providers.php`.
-
-```php
-<?php
-$provider = $this->getParam('provider', '');
-?>
-<label for="oauth2<?php echo $this->escape(ucfirst($provider)); ?>Appid">Application (Client) ID<span class="tooltip" data-tooltip="Provided by AzureAD"><i class="icon-info-circled"></i></span></label>
-<input name="appId" id="oauth2<?php echo $this->escape(ucfirst($provider)); ?>Appid" type="text" autocomplete="off" data-ls-bind="{{console-project.provider<?php echo $this->escape(ucfirst($provider)); ?>Appid}}" placeholder="Application ID" />
-<?php /*Hidden input for the final secret. Gets filled with a JSON via JS. */ ?>
-<input name="secret" data-forms-oauth-custom="<?php echo $this->escape(ucfirst($provider)); ?>" id="oauth2<?php echo $this->escape(ucfirst($provider)); ?>Secret" type="hidden" autocomplete="off" data-ls-bind="{{console-project.provider<?php echo $this->escape(ucfirst($provider)); ?>Secret}}" />
-<!-- [Your custom form inputs go here] -->
-```
-
-2. Add the config for creating the JSON in `public/scripts/views/forms/oauth-custom.js` using this template
-
-```js
-{
-    "[Provider]":{
-        "[JSON property name 1]":"[html element Id 1]",
-        "[JSON property name 2]":"[html element Id 2]"
-    }
-}
-```
```

Comment:
I believe custom HTML to support certain OAuth providers is still required. We might need @ArmanNik or @TGlide help to understand how its done in the new console

Reply:
If the OAuth does not require custom code then it's fine as it is. Otherwise the user need to create a new svelte component here https://github.com/appwrite/console/tree/main/src/routes/console/project-%5Bproject%5D/auth . The easiest way to handle this is to copy an existing "unique" OAuth and make the necessary changes

Reply:
I forgot to say that the user should still update the  `oauth-providers.ts` file by imported the newly created component and assigning it like this:
```js
import Microsoft from '../../routes/console/project-[project]/auth/microsoftOAuth.svelte';

...

  case 'microsoft':
      docs = 'https://developer.microsoft.com/en-us/';
      component = Microsoft;
      break;
```     

We had in mind to do a console repo clean-up/organizing so the location where we create new OAuth components might change 😊